time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -47,3 +48,15 @@ def __init__(self):
             SQLCRUDDao.__init__(self, TimeEntrySQLModel)
 
     return TimeEntriesSQLDao()
+
+
+container_definition = {
+    'id': 'time_entry',
+    'partition_key': PartitionKey(path='/tenant_id'),
+    'unique_key_policy': {
+        'uniqueKeys': [
+            {'paths': ['/owner_id', '/end_date']},
```

Comment:
These two fields belongs to the model. Why is this path different form the others to define uniqueKeys. ?

Reply:
Because here the uniqueness is the combination `owner_id` and `end_date`. This means that there cannot be 2 entries with the same `end_date`. Having into consideration that `null` is also a value, then if by mistake the client sends the creation of a time entry twice, as the second one will also have the end_date as null there will be a uniqueness issue and it will not be created. This way the client will be forced to close the existent opened time entry to create a new one.